integration_tests/test_model_download.py

```diff
@@ -68,6 +68,15 @@ def test_model_unversioned_succeeds(self):
         ]
         self.assert_files(actual_path, expected_files)
 
+    def test_download_private_model_succeeds(self):
+        actual_path = model_download("integrationtester/test-private-model/pyTorch/b0")
```

Comment:
all admins should also have access to the private model. This way, we don't all have to create a private model for this test.